aspnetcore/fundamentals/servers/aspnet-core-module/sample/Properties/PublishProfiles/publish-module.psm1

```diff
@@ -0,0 +1,1231 @@
+﻿# WARNING:  DO NOT MODIFY this file. Visual Studio will override it.
```

Comment:
@danroth27 should publish profiles be checked in with samples? They're automatically generated VS artifacts.